plugin/advisor/mysql/mysql_column_requirement_advisor_test.go

```diff
@@ -19,7 +19,7 @@ func TestColumnRequirement(t *testing.T) {
 					Status:  advisor.Warn,
 					Code:    common.NoRequiredColumn,
 					Title:   "Require columns",
-					Content: "\"book\" requires columns: id, created_ts, updated_ts, creator_id, updater_id",
+					Content: "\"book\" requires columns: created_ts, creator_id, id, updated_ts, updater_id",
```

Comment:
nit: shall we only print the columns that are missing instead of the ones in the policy?